code/modules/projectiles/projectile/beams.dm

```diff
@@ -55,6 +55,7 @@
 /obj/item/projectile/beam/xray
 	name = "\improper X-ray beam"
 	icon_state = "xray"
+	flag = "rad"

```

Comment:
this isn't a define?

Reply:
Elaborate.

Reply:
It makes it roll for rad armour, just as flag = "laser" rolls for laser armour.

Reply:
"rad" is not defined for the purpose of `flag`?

Reply:
I mean I've tested it, it works, so it must be.

Reply:
nvm this var is stupid